crates/bitwarden-exporters/src/csv.rs

```diff
@@ -0,0 +1,266 @@
+use std::collections::HashMap;
+
+use csv::Writer;
+use serde::Serializer;
+use thiserror::Error;
+use uuid::Uuid;
+
+use crate::{Cipher, CipherType, Field, Folder};
+
+#[derive(Debug, Error)]
+pub enum CsvError {
+    #[error("CSV error")]
+    Csv,
+}
+
+pub(crate) fn export_csv(folders: Vec<Folder>, ciphers: Vec<Cipher>) -> Result<String, CsvError> {
+    let folders: HashMap<Uuid, String> = folders.iter().map(|f| (f.id, f.name.clone())).collect();
+
+    let rows = ciphers
+        .iter()
```

Comment:
If we change this to `.into_iter()`, we can remove some clones below, in name, notes and fields at least